.lintstagedrc.json

```
@@ -0,0 +1,8 @@
+{
+  "!(.github/workflows/*.yml|*.js|package.json)": [
+    "prettier --ignore-unknown --write"
+  ],
+  ".github/workflows/*.yml": ["npx .", "prettier --write"],
```

I would include all yml files under `*yml`

```suggestion
  "!(.github/*/*.yml|*.js|package.json)": [
    "prettier --ignore-unknown --write"
  ],
  ".github/*/*.yml": ["npx .", "prettier --write"],
```

to cover yourself if you ever add a local action in this repo

---

My preference is to keep this as simple as possible. In the unlikely scenario that custom actions are added to the repo, we could do that.